src/styles/main.scss

```diff
@@ -1,3 +1,75 @@
-body {
-  margin: 0;
+@import "./normalize.scss";
+@import "./utilis/variables";
+@import "./utilis/mixins";
+@import "./utilis/extends";
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  gap: 1px;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  width: (7 * $cell-size) + (6 * $gap-size);
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    box-sizing: border-box;
```

Comment:
Better to set this property for all elements(*) and don't change without extra need